modules/my-cpp.el

```diff
@@ -0,0 +1,18 @@
+;;; package --- Summary
+
+;;; Commentary:
+
+;;; Code:
+(use-package lsp-mode :commands lsp)
+(use-package lsp-ui :commands lsp-ui-mode)
+(use-package company-lsp :commands company-lsp)
+
+(use-package ccls
+  :hook ((c-mode c++-mode objc-mode cuda-mode) .
+         (lambda () (require 'ccls) (lsp))))
+(setq lsp-diagnostic-package nil)
+(setq-default flycheck-disabled-checkers '(c/c++-clang c/c++-cppcheck c/c++-gcc))
+(setq ccls-executable "/path/to/ccls/Release/ccls")
```

Comment:
😅 Don't think this will work

Reply:
You mean `(setq lsp-diagnostic-package nil)`? I ripped that from the ccls github, originally `(setq lsp-prefer-flymake nil)`, but emacs said that was deprecated and favored this.

Reply:
`(setq ccls-executable "/path/to/ccls/Release/ccls")` I'm quite sure this path doesn't exists on either of our systems